### DISCUSSION TITLE
Infinite to be actual inifnite rather than reverse back

### DISCUSSION THREAD

**Discussion by jamborski-dev** [express-labs/pure-react-carousel#397] [Ideas]
Hi there! It would be a great option to have the inifnite prop make slider an actual infnite slider rather than a loop with a sudden reverse jump to the the first slide. With not too many slides it doesn't cause any problem, however if there is 10-20+ slides it can be a bit unpleasent visually IMHO

**Comment by AndyOooh** [comment]
I came here to ask the same. Does anyone have an idea if/how I can customize for this behavior?
